lib/publiccloud/aws_client.pm

```diff
@@ -75,8 +75,13 @@ sub init {
 
     if (get_var('PUBLIC_CLOUD_CREDENTIALS_URL')) {
         my $data = get_credentials('aws.json');
-        $self->key_id($data->{access_key_id});
-        $self->key_secret($data->{secret_access_key});
+        if (get_var('PUBLIC_CLOUD_SLES4SAP')) {
```

Comment:
can you please try to run VR only with that part of changes ? I suspect that this is only what you actually need

Reply:
It worked with the new VR. Thanks!